scripts/CpUtil.lua

```diff
@@ -376,4 +376,27 @@ function CpUtil.isStateOneOf(myState, states)
         end
     end
     return false
+end
+
+-- Return all fruitTypeIndexes where a fruitType needs preparation. 
+--
+-- Preparation marks an earth fruit since this is the only factor that diversifies them from normal fruit types
+-- SUGARCANE is an exception and ignored
+function CpUtil.getAllEarthFruits()
```

Comment:
Can we call it root vegetable instead of earth fruit? Earth fruit is not used in English.

Reply:
Also replacing all other occurrences of earth fruit?